api/env.template

```diff
@@ -24,6 +24,8 @@ ALLOWED_HOSTS=172.17.0.1,host.docker.internal
 #THUMBNAIL_JPG_QUALITY=80
 #THUMBNAIL_PNG_COMPRESSION=6
 
+#THUMBNAIL_TIMEOUT_PREFIX=thumbnail_timeout
```

Comment:
This value leaves off the separator, but the default in the `settings.py` has it. Should we be consistent here? Do we need to include the separator in the prefix or is that something we could inline in the template string used to generate the key: https://github.com/WordPress/openverse-api/pull/982/files#diff-be5fb4e5665130b04ebed93a52ff1730d80ffb734606e9f5c41dfb066c39bbcbR218

I don't have a strong opinion here in either direction, just wanting to be consistent with the settings.

Reply:
Resolved in 63dda22.